source/index.html.md

```diff
@@ -1,239 +1,69 @@
 ---
-title: API Reference
+title: Graasp Get Started With React
 
 language_tabs: # must be one of https://git.io/vQNgJ
   - shell
-  - ruby
-  - python
-  - javascript
 
 toc_footers:
   - <a href='#'>Sign Up for a Developer Key</a>
   - <a href='https://github.com/lord/slate'>Documentation Powered by Slate</a>
 
-includes:
-  - errors
-
 search: true
 ---
 
 # Introduction
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+Welcome to the `Graasp Get Started with React App`! An initial project structure.
+This `Get Started React App` has been developed for starters adoption. You can use it as much as you need, add small to big modifications. All links in the sections and this documentation will help you get started.
 
-We have language bindings in Shell, Ruby, Python, and JavaScript! You can view code examples in the dark area to the right, and you can switch the programming language of the examples with the tabs in the top right.
+Also, we will help you well understand the building blocks of React apps: elements and components.
 
-This example API documentation page was created with [Slate](https://github.com/lord/slate). Feel free to edit it and use it as a base for your own API's documentation.
+You can view code examples in the dark area to the right.
 
-# Authentication
+This guide is separated into several parts: <br>
+- Forking the initial repository <br>
+- Installing dependencies & Starting the application <br>
+- Setting linting `eslint-plugin-react` <br>
+- Testing the App
+- Adding your own dependencies <br>
+- Updating dependencies
 
-> To authorize, use this code:
+# Lab
```

Comment:
After `# Lab` you should have a subsection titled `React` and put the rest of the subsections underneath that. That way we can then have another one called `Angular`.